app/components/ui/UserWay.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const UserWay = ({ appID }) => {
+  const [installed, setInstalled] = useState(false);
+
+  useEffect(() => {
+    if (installed) return;
```

Comment:
This isn't the safest way of checking whether the script has been installed, since the state of the component gets blown away if the component is ever unmounted and then remounted. Since you're modifying the actual DOM, you should let that be the source of truth, so how about you set an `id` on the `<script>` element you're creating and inserting into the DOM and the check whether an element of that ID exists here?

Reply:
Cool, that works, updated!